codespell_lib/data/dictionary.txt

```diff
@@ -24899,6 +24971,8 @@ physisist->physicist
 phython->python
 phyton->python
 phy_interace->phy_interface
+piar->pair, plier, pier,
+piars->pairs, pliars, piers,
```

Comment:
1. Some reputable dictionaries considered `plier` as invalid, and it is written in plural form `pliers`.
2. Suggested word `pliars` is an invalid word.
* https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=plier
```text
piar->pair, pier, pliers,
piars->pairs, piers, pliers,
pliars->pliers
```